src/SFA.DAS.FAA.Web/Models/Apply/ApplicationSummaryViewModel.cs

```diff
@@ -0,0 +1,175 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.FAA.Application.Queries.Apply.GetApplicationSummary;
+using SFA.DAS.FAA.Domain.Enums;
+
+namespace SFA.DAS.FAA.Web.Models.Apply;
+
+public class ApplicationSummaryViewModel
+{
+    public Guid ApplicationId { get; set; }
+
+    [BindProperty]
+    public bool IsConsentProvided { get; set; }
+    public CandidateDetailsSection Candidate { get; init; } = new();
+
+    public static implicit operator ApplicationSummaryViewModel(GetApplicationSummaryQueryResult source)
+    {
+        return new ApplicationSummaryViewModel
+        {
+            Candidate = source.Candidate,
+            ApplicationQuestions = source.ApplicationQuestions,
+            DisabilityConfidence = source.DisabilityConfidence,
+            EducationHistory = source.EducationHistory,
+            InterviewAdjustments = source.InterviewAdjustments,
+            WorkHistory = source.WorkHistory,
+            IsDisabilityConfident = source.IsDisabilityConfident,
+        };
+    }
+
+    public class CandidateDetailsSection
+    {
+        public Guid Id { get; set; }
+        public string? GovUkIdentifier { get; set; }
+        public string? Email { get; init; }
+        public string FullName => $"{FirstName} {LastName}";
+        private string? FirstName { get; init; }
+        public string? MiddleName { get; set; }
+        private string? LastName { get; init; }
+        public string? PhoneNumber { get; init; }
+        private DateTime? DateOfBirth { get; init; }
+        public string BirthDate => DateOfBirth.HasValue ? DateOfBirth.Value.ToString("dd MMM yyyy") : string.Empty;
```

Comment:
Can we just double check that we should be showing this - I know we capture but I don't think its part of the application.